sklearn/linear_model/coordinate_descent.py

```diff
@@ -474,7 +474,8 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
             warnings.warn('Objective did not converge.' +
                           ' You might want' +
                           ' to increase the number of iterations.' +
-                          ' Fitting data with alpha near zero, e.g., 1e-8,' +
+                          ' Fitting float32 data with alpha near zero,' +
```

Comment:
Remove any specific value. Just say near zero

On 25 August 2016 at 23:34, Yen <notifications@github.com> wrote:

> In sklearn/linear_model/coordinate_descent.py
> <https://github.com/scikit-learn/scikit-learn/pull/6913#discussion_r76242986>
> :
>
> > @@ -474,7 +474,8 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
> >              warnings.warn('Objective did not converge.' +
> >                            ' You might want' +
> >                            ' to increase the number of iterations.' +
> > -                          ' Fitting data with alpha near zero, e.g., 1e-8,' +
> > +                          ' Fitting float32 data with alpha near zero,' +
>
> I have to admit that I'm not very sure about *all* the factors that will
> cause convergence issue, and thus not dare to determine a specific
> reference value.
>
> Or we can remove reference to the alpha value?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/6913/files/82fdf0962e3c7b0965b54ca137a56ab6d01fc226..c032d3b5820b53fb0717008435a44245cdb746f1#r76242986>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEz65p1hO8jA-O7s1Ka5BIexYN_p7qlks5qjZnKgaJpZM4I6SId>
> .
>